committee-code-of-conduct/governance/incident-process.md

```diff
@@ -0,0 +1,111 @@
+
+## Incident report workflow 
+ 
+### Initial triage 
+
+The Code of Conduct Committee responds to all emails in a timely manner, usually within a few days.
+
+### Recusal 
+
+Before beginning investigation on an incident, members can recuse (or refuse to pass judgement on) an incident if they feel a relationship with someone in the incident may hinder impartiality or create a perception of impropriety with respect to individuals involved in the reported incident. Code of Conduct Committee Members might recuse for the following reasons:
+
+- Direct reporting relationships, or company work relationships 
+- Close working relationships in the Kubernetes community, for example co-leading a SIG with the reporter or someone else mentioned in the report
+
+
+If all members of the Code of Conduct Committee felt the need to recuse themselves from an incident, the incident would be handled by our thid party mediator.
+
+### Reaching out to involved parties
+
+The Code of Conduct Committee will privately discuss the incident report, and may or may not decide that we need more information prior to determining whether to take any action.
+
+We consider the following at this stage:
+
+- Do we need clarification from the reporter beyond the initial report?
+- Do we need clarification from other individuals who may have been involved in, or witnesses to, the incident? 
+- Is there a public record of the incident which we can review, such as a chat log or video recording? 
+- Are there any privacy or safety considerations that we must take into account? For example, if we reach out to an individual named in the report, could this jeapordize the safety of the reporter or other individuals?
+
+It is our intention to put as little emotional labor on those who have been harmed as possible, and to protect the safety (both physical and emotional) of all community members. We labor to be supportive and non-judgemental and to make the reporting process as safe and low anxiety as possible.
+
+In all instances these clarifying discussions are confidential.
+
+
+
+## Incident response workflow
+
+### Reconvening the Committee
+
+When we have more information, the Code of Conduct Committee reconvenes, shares all information gathered, and moves on to incident response. 
+
+Depending on the complexity and severity of the incident, reaching a consensus may take some time. It may require follow up conversations with affected individuals, or other inquiries.
+
+### Deciding on a Course of Action 
+
+We do not act recklessly, and in deciding on a course of action, we work as a team to include diverse perspectives, support the immediate safety needs of our community members, and support the long-term health of this community.
+
```

Comment:
a link to "trauma-informed restorative justice framework" resources may be helpful to the reader?

Reply:
That's  great feedback,  @dims . I don't have a public link handy, but I can look for some.

A portion of our knowledge on this comes from private (paid for by LF) training we've received. Do you think a link to those courses would be helpful, even if we can't make them available to everyone?

Reply:
correct @AevaOnline that sounds great!